ports/kf5notifications/vcpkg.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "kf5notifications",
+  "version-semver": "5.84.0",
```

```suggestion
  "version": "5.84.0",
```
This might be `version `here.

---

Done.